src/main/java/Parser.java

```diff
@@ -0,0 +1,37 @@
+public class Parser {
+
+    public static Command parse (String input) {
+        String[] inputSplit = input.split(" ");
+        if (input.equals("bye")) {
+            return new ExitCommand();
+        } else if (input.equals("list")) {
+            return new DisplayListCommand();
+        } else if (inputSplit[0].equals("delete")) {
+            int inputIndex = Integer.parseInt(inputSplit[1]);
+            int actualIndex = inputIndex - 1;
```

Comment:
I really like how you split the different processing of the commands into temp variables which makes it very easy to follow!!! :)

Reply:
Thank you! Hope they were as helpful to me as they were to you :)